pkg/rpc/rpc_config.go

```diff
@@ -0,0 +1,26 @@
+package rpc
```

Comment:
I think it's a server config, thus belonging to the server package.

Reply:
Moving `rpc_config.go` to server package leads to the import cycle (package `config` uses package `server` and `server_helper_test` from `server` package uses package `config`). What should we do in this case?

Reply:
Is it possible to move tests to `server_test` package? That would be a nice solution. If it's not possible then we can leave this config as is in the `rpc` package.

Reply:
That's not possible, because server_helper_test.go uses unexported method from server package - requestHandler(...)